analyses/ps3_script.py

```diff
@@ -353,4 +353,47 @@ def lorenz_curve(y_true, y_pred, exposure):
     )
 )
 
+# %%
+# PS4 EX2
+import lightgbm as lgb
```

Comment:
Can you put import change to the beginning of the file in the section where we group all the imports?